macOS/DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionNavBarPopoverManager.swift

```diff
@@ -213,31 +216,56 @@ final class NetworkProtectionNavBarPopoverManager: NetPPopoverManager {
                 activeSitePublisher: activeSitePublisher,
                 uiActionHandler: uiActionHandler)
 
-            let statusViewModel = NetworkProtectionStatusView.Model(controller: controller,
-                                              onboardingStatusPublisher: onboardingStatusPublisher,
-                                              statusReporter: statusReporter,
-                                              uiActionHandler: uiActionHandler,
-                                              menuItems: { [weak self] in
-
+            let menuItems = { [weak self] () -> [NetworkProtectionStatusView.Model.MenuItem] in
                 guard let self else { return [] }
 
                 guard featureFlagger.isFeatureOn(.networkProtectionAppExclusions) else {
                     return legacyStatusViewSubmenu()
                 }
 
                 return statusViewSubmenu()
-            },
-                                              agentLoginItem: LoginItem.vpnMenu,
-                                              isMenuBarStatusView: false,
-                                              userDefaults: .netP,
-                                              locationFormatter: DefaultVPNLocationFormatter(),
-                                              uninstallHandler: { [weak self] in
-                _ = try? await self?.vpnUninstaller.uninstall(removeSystemExtension: true)
-            })
+            }
+
+#if APPSTORE
+            let isExtensionUpdateOfferedPublisher: CurrentValuePublisher<Bool, Never> = {
+                let initialValue = featureFlagger.isFeatureOn(.networkProtectionAppStoreSysexMessage)
+                    && !vpnAppState.isUsingSystemExtension
+
+                let publisher = vpnAppState.isUsingSystemExtensionPublisher
+                    .map { [featureFlagger] value in
+                        featureFlagger.isFeatureOn(.networkProtectionAppStoreSysexMessage) && !value
+                    }.eraseToAnyPublisher()
+
+                return CurrentValuePublisher(initialValue: initialValue, publisher: publisher)
+            }()
+#else
+            let isExtensionUpdateOfferedPublisher = CurrentValuePublisher(initialValue: false, publisher: Just(false).eraseToAnyPublisher())
```

Comment:
This publisher tells the VPN status view whether the VPN update is offered to users or not.  For `APPSTORE` builds it checks if the user has a system extension.

For other builds this is always `false` as the VPN update is never offered.